src/search/search_tables.h

```diff
@@ -109,10 +111,21 @@ class SearchTables {
         m_historyMoves.update(board, move, ply);
     }
 
+    inline movegen::Move getCounterMove(const movegen::Move& prevMove) const
```

Comment:
copy is cheaper (16 bits)
```suggestion
    inline movegen::Move getCounterMove(movegen::Move prevMove) const
```